content/articles/2017-12-testthat-2-0.Rmd

```diff
@@ -0,0 +1,207 @@
+---
+title: testthat 2.0.0
+author: Hadley Wickham
+date: '2017-12-19'
+slug: testthat-2-0-0
+categories:
+  - package
+tags: [package, r-lib]
+photo:
+  url: https://unsplash.com/photos/Lu2pfy_8VKg
+  author: Cameron Kirby
+---
+
+```{r setup, include = FALSE}
+library(testthat)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are very excited to announce that testthat 2.0.0 is now available on CRAN! Testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily. Learn more at <http://r-pkgs.had.co.nz/tests.html>. Install the latest version with:
```

Comment:
Link "testthat" to <http://testthat.r-lib.org>?

Or, in any case, somehow position <http://testthat.r-lib.org> as a resource for the testthat package and <http://r-pkgs.had.co.nz/tests.html> as guidance on the testing process.